src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -152,13 +152,32 @@ private static void DetectRHEL6()
             }
         }
 
+        private static string GetOsReleaseFilePath()
```

Comment:
I would make it more atomic as the flow of release processing will be different for each OS

```suggestion
        private static string GetLinuxReleaseFilePath()
```

Reply:
Renamed, thanks!